src/components/20-molecules/datepicker/index.js

```diff
@@ -278,18 +309,21 @@ class AXADatepicker extends NoShadowDOM {
     style.width = getFormattedStyle(width); // set width to component's css
     const formattedHeight = getFormattedStyle(height); // set height to input-wrap element because of optional label
 
-    const cellClasses = ({ sameMonth, today, selected, inactive }) =>
-      classMap({
+    const cellClasses = ({ sameMonth, today, selected, inactive, value }) => {
```

Comment:
A comment would imho be helpful about what defines a selected cell. Somehow either if it is user selected, if it is preselected AND if it's 'today', right?

Reply:
OK, will do. It's selected iff the same day as the internal this._date.

Reply:
Even better action: remove reference to selected here, it's useless ;-)